doc/users/next_whats_new/engformatter_usetex_usemathtext.rst

```diff
@@ -0,0 +1,11 @@
+`EngFormatter` now accepts `usetex`, `useMathText` as keyword only arguments
+``````````````````````````````````````````````````````````````````````````````````````````````
+
+A public API has been added to `EngFormatter` to control how the numbers in the ticklabels will be rendered.
```

Comment:
Lines should not be longer than 79 chars - also for documentation source files. Please re-wrap.